### DISCUSSION TITLE
Warn if autorun or reaction does not access any observables?

### DISCUSSION THREAD

**Discussion by IanBellomy** [mobxjs/mobx#3964] [Ideas]
Made a mistake that led to a reaction being created before the item it watched was made observable. 

Was then a little surprised there wasn't some kind of check in reaction/autorun that at least one observable was accessed. 

If discoverable, it smells like a situation worth issuing a warning about?

**Comment by mweststrate** [comment]
see https://mobx.js.org/configuration.html#reactionrequiresobservable-boolean :)

**Reply by IanBellomy** [reply]
Ahhh , nice.
